crates/pixi_consts/src/consts.rs

```diff
@@ -19,7 +19,12 @@ pub const SOLVE_GROUP_ENVIRONMENTS_DIR: &str = "solve-group-envs";
 pub const PYPI_DEPENDENCIES: &str = "pypi-dependencies";
 pub const TASK_CACHE_DIR: &str = "task-cache-v0";
 pub const PIXI_UV_INSTALLER: &str = "uv-pixi";
+pub const CONDA_PACKAGE_CACHE_DIR: &str = "pkgs";
+pub const CONDA_REPODATA_CACHE_DIR: &str = "repodata";
```

Comment:
These values are also in [`rattler_cache`](https://docs.rs/rattler_cache/0.1.8/rattler_cache/) we should use those!

Reply:
Now using those, added `rattler_cache` as a dep to the main pixi and use it as a workspace dep in `pixi_consts`

Reply:
can we then get rid of these constants? or reexport them?